client/policy_validator.go

```diff
@@ -61,13 +59,12 @@ type ClusterPolicyValidator struct {
 	cli           *VanClient
 	dc            *discovery.DiscoveryClient
 	skupperPolicy v1alpha1.SkupperClusterPolicyInterface
-	labelRegex    *regexp.Regexp
+	mockEnabled   bool
```

Comment:
Myabe 'disablePolicyDiscovery' would be a nicer name for this?

Reply:
+1